lib/faktory_worker/connection_manager.ex

```diff
@@ -29,6 +29,11 @@ defmodule FaktoryWorker.ConnectionManager do
           do: send_command(%{state | conn: nil}, command, false),
           else: {error, state}
 
+      # Handle errors from Faktory that should not be tried again, such as
+      # unique jobs.
+      {{:error, "Halt: " <> _ = reason}, state} ->
```

Comment:
Not sure which is better but should we return the halt prefix here I wonder? It feels like returning `{:ok, "job not unique"}` would be less confusing to a user of the lib that doesn't understand what's going on under the covers.

I'm not sold on that just an idea that I wanted to put out there.

Reply:
I'm going to go with removing the "Halt: " part of the string and return just the message.

Reply:
Awesome sounds good 👍